pkgs/applications/version-management/gitlab/gitlab-shell/go-1_21.patch

```diff
@@ -0,0 +1,182 @@
+diff --git a/go.mod b/go.mod
+index 513ccc4..0ecbdd2 100644
+--- a/go.mod
++++ b/go.mod
+@@ -1,6 +1,8 @@
+ module gitlab.com/gitlab-org/gitlab-shell/v14
+ 
+-go 1.20
++go 1.21
++
++toolchain go1.22.3
```

Comment:
Those lines are not strictly required and could cause the build to fail when go bitches about them.

Reply:
Good to know. Fortunately, the patch is no longer needed.